modules/gdnative/register_types.cpp

```diff
@@ -208,6 +210,7 @@ void unregister_gdnative_types() {
 	singleton_gdnatives.clear();
 
 	unregister_nativearvr_types();
+	unregister_pluginscript_types();
 	unregister_nativescript_types();
```

Comment:
Oh while you're here, can you make the order of unregistering the opposite of registering? @BastiaanOlij put the nativearvr stuff in the wrong order 😛

Reply:
done.